### DISCUSSION TITLE
Jetbrains copilot

### DISCUSSION THREAD

**Discussion by yimaiyan** [community/community#10866] [Copilot Conversations] [answered]
What if I have email A which own the copilot license but my jetBrains account is registered by B email?  
When I am trying to log in github account, it will automatically redirect to email B( which doesn't have copilot license).  
How can I switch emails when trying to use copilot in IDEA?

**Comment by jansorg** [comment, marked as answer]
Copilot uses your GitHub account and not your JetBrains account. Do you perhaps have to different GitHub accounts?
If that's the case, try to sign in with the GitHub account which has permissions for Copilot before you log in to GitHub Copilot.
If that doesn't help, please provide more context how this is related to your JetBrains account.
